### ISSUE TITLE
Feature Request : Add code snippet in Story

### ISSUE THREAD

**Issue by quentin7b** [ookami-kb/storybook_flutter#57]
First of all, thank your for this project.

It is easy to use, and nice to see.

Here is a feature request we might like for upcoming updates : **Add a possibility to add a code snippet on a component Story**

It would be great to see a minimum code to use to implement the demonstrated widget.

**Comment by luvetter** [comment]
I was thinking about something similar. We could provide these code snippets as md files. These could also be used with @example in the dartdoc (or we could reuse existing md files). We would then create a plugin for the storybook that uses [flutter_highlight](https://pub.dev/packages/flutter_highlight) to display the code snippets. 

I'm just unsure how to assign md file(s) to a story.

And this should probaly wait, until the rework of the plugin api is finished.

**Comment by ookami-kb** [comment]
Yeah, makes sense to me. I was also thinking about some code generation for snippets etc – e.g. it can be taken from the docstrings.

**Comment by klevissx** [comment]
Any update on this, or any instruction on how it can be added to the lib. I would gladly help on creating a PR. @ookami-kb
